tests/post/features/pods_alive.feature

```diff
@@ -6,15 +6,15 @@ Feature: Pods should be alive
 
     Scenario: Expected Pods
         Given the Kubernetes API is available
-        Then we have at least 1 running pod labeled 'component=kube-controller-manager'
-        And we have at least 1 running pod labeled 'component=kube-scheduler'
-        And we have at least 1 running pod labeled 'component=kube-apiserver'
-        And we have at least 1 running pod labeled 'component=etcd'
-        And we have at least 1 running pod labeled 'k8s-app=kube-proxy'
-        And we have at least 1 running pod labeled 'k8s-app=calico-node'
-        And we have at least 2 running pod labeled 'k8s-app=kube-dns'
-        And we have at least 1 running pod labeled 'app=salt-master'
-        And we have at least 1 running pod labeled 'app=repositories'
+        Then we have 1 running pod labeled 'component=kube-controller-manager' in the 'kube-system' namespace
```

Comment:
I did something similar here: https://github.com/scality/metalk8s/pull/1031/commits/77104877232d410e1048e7259d1f2d3c7a6869c0

Maybe we should synchronize to try to have something similar.

Reply:
Yep, maybe I can rebase on your PR and adapt my changes to fit with yours ?

Reply:
Let's rebase the other way around